scipy/spatial/transform/tests/test_rotation_groups.py

```diff
@@ -0,0 +1,152 @@
+from __future__ import division, print_function, absolute_import
+
+import pytest
+
+import numpy as np
+from numpy.testing import assert_array_almost_equal
+import itertools
+from scipy.spatial.transform import Rotation
+from scipy.optimize import linear_sum_assignment
+from scipy.spatial.distance import cdist
+from scipy.constants import golden as phi
+
+
+tol = 1E-12
```

Comment:
Just a small thing --- it is recommended to use uppercase for module-level constants.

Reply:
Thanks.